modules/uhttp/client/client_middleware_test.go

```diff
@@ -103,7 +107,8 @@ func TestExecutionChainOutboundMiddleware_AuthContextPropagation(t *testing.T) {
 func TestExecutionChainOutboundMiddleware_AuthContextPropagationFailure(t *testing.T) {
 	withOpentracingSetup(t, auth.FakeFailureClient, func(tracer opentracing.Tracer) {
 		execChain := newExecutionChain(
-			[]OutboundMiddleware{authenticationOutbound(fakeAuthInfo{_testYaml})}, contextPropagationTransport{t},
+			[]OutboundMiddleware{authenticationOutbound(fakeAuthInfo{_testYaml})},
```

Comment:
maybe not worth it now, but this seems like it could be DRY'd up to make it more readable/maintainable, since this is pretty much exactly the same (or is exactly the same?) as lines 94 and 95